mailmerge.py

```diff
@@ -152,10 +160,19 @@ def merge(self, parts=None, **replacements):
 
     def __merge_field(self, part, field, text):
         for mf in part.findall('.//MergeField[@name="%s"]' % field):
-            mf.clear()
+            children = list(mf)
+            mf.clear()  # clear away the attributes
             mf.tag = '{%(w)s}r' % NAMESPACES
-            mf.append(Element('{%(w)s}t' % NAMESPACES))
-            mf[0].text = text
+            mf.extend(children)
+
+            text_node = Element('{%(w)s}t' % NAMESPACES)
+            text_node.text = text
+
+            ph = mf.find('MergeText')
+            if ph is not None:
```

Comment:
Could we add a comment here? It is not quite obvious why there is a if else as the code which makes this distinction is in a different function.